packages/eds-core-react/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -50,27 +51,25 @@ const Placeholder = styled.div`
 `
 
 export const Default: Story<DialogProps> = (args) => {
-  //  Note: This example is not interactive, as Storybook
-  // doesn't yet support to manipulate subcomponents via Storybook Args
   const { open, isDismissable } = args
-  const [isOpen, setIsOpen] = useState(open)
+  const [, updateArgs] = useArgs()
   const handleClose = () => {
-    setIsOpen(false)
+    updateArgs({ open: false })
   }
   return (
     <>
-      <Button aria-haspopup="dialog" onClick={() => setIsOpen(true)}>
+      <Button aria-haspopup="dialog" onClick={() => updateArgs({ open: true })}>
```

Comment:
Add a `handleOpen` function? To be concise with our other stories.

Reply:
Ok added `handleOpen `and `handleClose `to all the dialog stories